### DISCUSSION TITLE
Why the limit?

### DISCUSSION THREAD

**Discussion by Zibri** [coqui-ai/TTS#3320] [General Q&A] [answered]
I tried to generate a file in italian language.
I got this warning/error:

[!] Warning: The text length exceeds the character limit of 213 for language 'it', this might cause truncated audio.

Why is there a limit if the process is on the local machine?

**Comment by erogol** [comment, marked as answer]
because model cannot processes longer text.

**Reply by catbears** [reply]
I ran into that as well. I do the lazy method like this:

```bash
$ tts --model_name tts_models/multilingual/multi-dataset/xtts_v2 --out_path output/my_wav.wav --text "$(cat my_file.txt)" --speaker_idx "Viktor Eka" --language_idx en
```

Do I need to write a Python script to handle text length? (is this command line behaviour intentional?)

**Reply by kubinka0505** [reply]
Splitting may work but its bad practice

**Reply by esrasaif** [reply]
is it right to fine tunning the model to make the length longer or split the text and generate chuncks of the voice then connecting them ??
